Python/Python-tokenize.c

```diff
@@ -240,7 +240,11 @@ tokenizeriter_next(tokenizeriterobject *it)
             type = NAME;
         }
         else if (type == NEWLINE) {
-            str = PyUnicode_FromString("\n");
+            if (it->tok->start[0] == '\r') {
+                str = PyUnicode_FromString("\r\n");
```

Comment:
Byw, I think we may be leaking the previous `str` (we are missing a `XDECREF` here)

Reply:
fixed: d3d4ff5fd74338b143c04befb4e7214f05b3ff55